src/main/java/io/cdap/plugin/salesforce/SalesforceConnectionUtil.java

```diff
@@ -42,6 +42,22 @@ public static PartnerConnection getPartnerConnection(AuthenticatorCredentials cr
     return new PartnerConnection(connectorConfig);
   }
 
+  /**
+   * Creates {@link AuthenticatorCredentials} instance based on given parameters.
+   *
+   * @param username Salesforce username
+   * @param password Salesforce password
+   * @param consumerKey Salesforce consumer key
+   * @param consumerSecret Salesforce consumer secret
+   * @param loginUrl Salesforce authentication url
+   * @return authenticator credentials
+   */
+  public static AuthenticatorCredentials getAuthenticatorCredentials(String username, String password,
```

Comment:
Looks like the compile error is fixed in another PR - https://github.com/data-integrations/salesforce/pull/96.
I think it would be easier if we merge that and just fix the bulk close error in this PR - avoid merge conflicts when we cherrypick to release/1.3 branch.